zone.go

```diff
@@ -62,6 +62,9 @@ type Zone struct {
 	// URL to look up whois info for a subdomain of this Zone
 	whoisURL string
 
+	// URLs for RDAP endpoints for a subdomain of this Zone
```

Comment:
@ydnar If I've removed the Parent walking stuff, is this comment about "subdomain of this Zone" accurate? If not, what should it be?

Reply:
That comment is fine, because it pertains to direct subdomains of the zone.